src/stores/pacbioLibraries.js

```diff
@@ -150,7 +152,6 @@ export const usePacbioLibrariesStore = defineStore('pacbioLibraries', {
     async deleteLibraries(libraryIds) {
```

Comment:
Libraries should not be destroyable in the UI so we should remove this and the associated components.

Reply:
I think libraries should be destroyable as long as they have not been used in a pool or a sequencing run. Also I am not sure whether we have an active status on libraries which could be another option.

Reply:
OK. Worth discussing. If they can be destroyable under certain conditions we will need a story for it. We will need to implement before this epic branch goes to prod because otherwise users can destroy libraries tied to runs which would break a lot of things.

Reply:
Could you add it to your epic branch. It is a minor change and we just need to check if either it belongs to a sequencing run or a pool? You can use the before_destroy hook.